executor/distsql_test.go

```diff
@@ -335,14 +363,18 @@ func (s *testSuite3) TestIndexLookUpStats(c *C) {
 		TableTaskNum:     2,
 		Concurrency:      1,
 	}
-	c.Assert(stats.String(), Equals, "index_task: {total_time: 5s, fetch_handle: 2s, build: 1s, wait: 2s}, table_task: {total_time: 2s, num: 2, concurrency: 1}")
-	c.Assert(stats.String(), Equals, stats.Clone().String())
+	require.Equal(t, "index_task: {total_time: 5s, fetch_handle: 2s, build: 1s, wait: 2s}, table_task: {total_time: 2s, num: 2, concurrency: 1}", stats.String())
+	require.Equal(t, stats.Clone().String(), stats.String())
 	stats.Merge(stats.Clone())
-	c.Assert(stats.String(), Equals, "index_task: {total_time: 10s, fetch_handle: 4s, build: 2s, wait: 4s}, table_task: {total_time: 4s, num: 4, concurrency: 2}")
+	require.Equal(t, "index_task: {total_time: 10s, fetch_handle: 4s, build: 2s, wait: 4s}, table_task: {total_time: 4s, num: 4, concurrency: 2}", stats.String())
 }
 
-func (s *testSuite3) TestIndexLookUpGetResultChunk(c *C) {
-	tk := testkit.NewTestKit(c, s.store)
+func TestIndexLookUpGetResultChunk(t *testing.T) {
+	t.Parallel()
```

Comment:
```t.Parallel()``` need to be removed. it is a decision that we have made recently.

Reply:
sorry , not familiar with some git operations , things seemed went bad , just close and create a new PR............